awx/main/models/rbac.py

```diff
@@ -33,29 +33,39 @@
 ROLE_SINGLETON_SYSTEM_AUDITOR='system_auditor'
 
 role_names = {
-    'system_administrator' : _('System Administrator'),
-    'system_auditor'       : _('System Auditor'),
-    'adhoc_role'           : _('Ad Hoc'),
-    'admin_role'           : _('Admin'),
-    'auditor_role'         : _('Auditor'),
-    'execute_role'         : _('Execute'),
-    'member_role'          : _('Member'),
-    'read_role'            : _('Read'),
-    'update_role'          : _('Update'),
-    'use_role'             : _('Use'),
+    'system_administrator': _('System Administrator'),
+    'system_auditor': _('System Auditor'),
+    'adhoc_role': _('Ad Hoc'),
+    'admin_role': _('Admin'),
+    'project_admin_role': _('Project Admin'),
+    'inventory_admin_role': _('Inventory Admin'),
+    'credential_admin_role': _('Credential Admin'),
+    'workflow_admin_role': _('Workflow Admin'),
+    'notification_admin_role': _('Notification Admin'),
+    'auditor_role': _('Auditor'),
+    'execute_role': _('Execute'),
+    'member_role': _('Member'),
+    'read_role': _('Read'),
+    'update_role': _('Update'),
+    'use_role': _('Use'),
 }
 
 role_descriptions = {
-    'system_administrator' : _('Can manage all aspects of the system'),
-    'system_auditor'       : _('Can view all settings on the system'),
-    'adhoc_role'           : _('May run ad hoc commands on an inventory'),
-    'admin_role'           : _('Can manage all aspects of the %s'),
-    'auditor_role'         : _('Can view all settings for the %s'),
-    'execute_role'         : _('May run the %s'),
-    'member_role'          : _('User is a member of the %s'),
-    'read_role'            : _('May view settings for the %s'),
-    'update_role'          : _('May update project or inventory or group using the configured source update system'),
-    'use_role'             : _('Can use the %s in a job template'),
+    'system_administrator': _('Can manage all aspects of the system'),
+    'system_auditor': _('Can view all settings on the system'),
+    'adhoc_role': _('May run ad hoc commands on an inventory'),
+    'admin_role': _('Can manage all aspects of the %s'),
+    'project_admin_role': _('Can manage all projects of the %s'),
+    'inventory_admin_role': _('Can manage all inventories of the %s'),
+    'credential_admin_role': _('Can manage all credentials of the %s'),
+    'workflow_admin_role': _('Can manage all workflows of the %s'),
+    'notification_admin_role': _('Can manage all notifications of the %s'),
+    'auditor_role': _('Can view all settings for the %s'),
+    'execute_role': _('May run the %s'),
```

Comment:
It doesn't make any sense to say

> May run the organization

but you got yourself into a corner by the way roles auto-generate help text, and the fact that you named the role the same thing on both organizations and JTs.